gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 kotlinVersion = "1.8.21"
 
 # Logging
-slf4jVersion = { strictly = "1.7.36" }
+slf4jVersion = "2.0.13"
```

Comment:
Does this not break compatibility for CorDapps ?

Reply:
No, it does not.